api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -184,6 +193,26 @@ COUNT(distinct p1.PERSON_ID) as count_value,COUNT(distinct p1.PERSON_ID) as sour
 on p1.person_id = co1.person_id where co1.condition_concept_id != co1.condition_source_concept_id
 group by co1.condition_source_concept_id, p1.gender_concept_id"
 
+# Condition gender identity
```

Comment:
Did you write the below SQL yourself or did it come from somewhere? Could you add a comment here summarizing what this is doing?

Reply:
i wrote this myself, this is similar to the one above (3101) except that this calculates the count of gender identity from the table created above unlike the gender counts calculated in 3101 from person bq table.